_docs/_latest/concepts/forseti-on-gke.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Forseti on GKE
+order: 006
+---
+
```

Comment:
Should remember to include the beta-release-feature here: https://github.com/forseti-security/forseti-security/blob/forsetisecurity.org-dev/_includes/docs/latest/beta-release-feature.md